lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -612,18 +611,14 @@ static void libjit_max_pool_xy_generic(const T *inW, T *outW, size_t *inXY,
               if (first || (val >= max)) {
                 first = 0;
                 max = val;
-                maxX = ox;
-                maxY = oy;
+                argmaxNHWC =
+                    ((n * inWdims[1] + ox) * inWdims[2] + oy) * inWdims[3] + z;
```

Comment:
nit: This is the same as `libjit_getXYZW(inWdims, n, ox, oy, z)`, you can just call that instead.